neo/Network/P2P/RemoteNode.cs

```diff
@@ -142,6 +142,16 @@ private void OnSend(IInventory inventory)
             if (Version?.Relay != true) return;
             if (inventory.InventoryType == InventoryType.TX)
             {
+                if (inventory is Transaction tx && tx.Version >= 2)
+                {
+                    // Check PoW spam protection for drop TX
+
+                    if (tx.Difficult > Settings.Default.TransactionDifficult)
```

Comment:
I think that we don't need to establish a difficulty, we just compare user transactions... the ones starting with more zeroes get priority.